cg-spec/editors_draft.html

```diff
@@ -1751,115 +1814,353 @@ <h2>Embedded Triple Expression</h2>
           <code>&lt;&lt;S P O&gt;&gt;</code>, where <code>S</code>, 
           <code>P</code> and <code>O</code> conform to
           <a href="#rExprVarOrTerm">ExprVarOrTerm</a>,
-          evaluates function <code>TRIPLE(S, P, O)</code>.
+          is evaluated as <code>TRIPLE(S, P, O)</code>.
         </p>
       </section>
 
-      <section class="informative">
-        <h2>Examples and Discussion</h2>
+      <section id="rdf-star-func-sameTerm">
+        <h3>sameTerm</h3>
+        <p class="box">
+          <code>xsd:boolean</code>
+          <code style="color:black;font-weight:bold">sameTerm</code>
+          (<code style="color:black">term</code>, <code style="color:black">term</code>)
+        </p>
+        
+        <p>The function <code>sameTerm</code> is extended to support term comparison for
+          RDF-star triple terms by defining:
+        </p>
+        <div class="box">
+          <p>
+            If both arguments are RDF-star triple terms:
+          <p style="margin-left:20px">
+            <code style="color:black;font-weight:bold">sameTerm</code><code>(&lt;&lt;S1 P1 O1&gt;&gt;,
+              &lt;&lt;S2 P2 O2&gt;&gt;)</code>
+          </p>
+          <p>
+            is evaluated as
+          </p>
+          <p style="margin-left:20px">
+            <code>sameTerm(S1, S2) && sameTerm(P1, P2) && sameTerm(O1, O2)</code>
+          </p>
+          <p>This is applied recursively.</p>
+          <p>If one argument is an RDF-star triple term and the other argument is not, 
+            the result is <code>false</code>.
+          </p>
+        </div>
+      </section>
+      
+      <section id="sparql-compare">
+        <h3>RDFstar-compare</h3>
+        
+        <p>The function <code>RDFstar-compare</code> is added in support of value-comparison
+          operators such as <code>=</code> and for <code>ORDER BY</code>.
+        </p>
+        <p class="box">
+          <code>xsd:boolean</code> &nbsp;
+          <code style="color:black;font-weight:bold">RDFstar-compare</code>
+          (<code style="color:black">term</code>, <code style="color:black">term</code>)
+        </p>
+        <p>The function <code>RDFstar-compare</code> returns -1,0,1, or throws an error,
```

Comment:
```suggestion
        <p>The function <code>RDFstar-compare</code> returns `-1`, `0`, or `1`, or throws an error,
```
(Respec replaces the backticks with `<code>` tags.)

Reply:
Done (to combine with other work in-progress, and not have lots to small commits, done on my local copy).

It's not offering "batch" to me.